lib/elastic/app-search/client/credentials.rb

```diff
@@ -0,0 +1,30 @@
+# Synonyms API - https://swiftype.com/documentation/app-search/api/analytics
+module Elastic
+  module AppSearch
+    class Client
+      module Credentials
+
+        # Retrieve available credentials
+        def list_credentials(current: 1, size: 20)
+          get("credentials", :page => { :current => current, :size => size })
+        end
+
```

Comment:
Looks like getOneKey method is missing https://swiftype.com/documentation/app-search/api/credentials#single

Reply:
Added in https://github.com/elastic/app-search-ruby/pull/16/commits/9c5273ede4a05e428d6b89c1914d9744a8e18c4e